src/localization/bundles/en.json

```diff
@@ -553,6 +553,20 @@
     "server.replace.prepare.failure": "Failed to prepare for replacing server",
     "update.commit.failure": "Failed to commit update changes. {0}",
     "install.progress.failure": "Failed to install SLES OS",
+    "server.retrieve.discovered.servers.error": "Failed to retrieve discovered servers. {0}",
+    "server.retrieve.manual.servers.error": "Failed to retrieve manually added servers. {0}",
+    "server.validating": "Validating Changes...",
+    "server.addserver.prepare": "Preparing Adding Servers in Progress",
+    "server.addserver.prepare.failure": "Failed to prepare for adding servers",
+    "server.addserver.deploy.failure": "Failed to deploy newly added servers",
+    "server.addserver.compute.deploy": "Deploying Compute Servers in Progress",
+    "server.addserver.deploy.confirm": "The deployment for new servers will take a while. It will validate and commit data and run playbooks.  \n\n Are you sure you want to proceed? Select Yes to proceed.",
+    "server.deploy.progress.wipe-disks": "Wipe Disks",
+    "server.deploy.progress.gen-hosts-file": "Generate Host Files",
+    "server.deploy.progress.addserver.deploy": "Deploy New Servers",
+    "server.deploy.progress.update-monasca": "Update Monasca",
+    "server.deploy.addserver.complete": "Successfully Added Compute Servers",
+    "server.addserver.validate.error.title": "Failure of validating newly added servers",
     "server.addserver.error.get-deployed-servers": "Failed to get deployed servers. {0}",
     "server.addserver.wipedisk.message": "This will set to wipe disk for all newly added servers"
```

Comment:
not part of this commit, but this wording is not quite right

Reply:
can you be more specific? which one need fix?